doc/quickstart.rst

```diff
@@ -18,10 +18,249 @@ examples below:
 	example so that you can quickly get up to speed with no external
 	dependencies.
 
-Generating a Live Report
-------------------------
+``iotile-analytics`` works on Python 2.7+ and Python 3.5+.  There is no usage difference
+on Python 2 or Python 3.
+
+
+Generating Live Reports
+-----------------------
 
 Live Reports are a key part of IOTile Analytics.  They are static HTML documents
 that either embed data and plots or reference local data files that can be
 accessed without needing a web server.
 
+.. important::
+
+	Download :download:`freezer.hdf5 <data/freezer.hdf5>` to use as source data and
+	place it in your current working directory.
+
+The quickest way to generate a live report is to use the `analytics-host` command
+line program that comes with `iotile-analytics-interactive`.  You can use the 
+program to generate reports from datasets that are either stored locally on your
+computer or downloaded on the fly from iotile.cloud.
+
+Almost always, the output of `analytics-host` is an interactive html+javascript
+document embedding graphs, data tables and other widgets to let you scroll through
+your data.  Sometimes the output is just plain text or a different binary format.
+
+Python packages you have installed can register LiveReport types with
+`analytics-host` so that you run them from the command line.  Let's see what
+reports you have installed:
+
+.. code:: bash
+
+	$ analytics-host -l
+	Installed Report Count: 3
+
+	Known Reports:
+	 - save_hdf5: Save all data locally as an HDF5 database file.
+	 - stream_overview: Display histogram and time series information about a stream.
+	 - basic_info: A basic LiveReport that just prints source information.
+
+
+You have 3 reports installed.  You can download data from iotile.cloud for fast local 
+access, you can print basic_info about a data source and you can create an interactive
+html report about a data stream.
+
+Let's look at the data source `freezer.hdf5` that you downloaded.
```

Comment:
Should the first example be either without the saved file, or the example showing how to download it?